### DISCUSSION TITLE
A different question about SKR mini E3 + JyersUI

### DISCUSSION THREAD

**Discussion by gileswells** [Jyers/Marlin#1155] [Q&A] [answered]
I've searched these discussions and the issues in this repo and others trying to come to the end of this question that I have. From what I can see it is _possible_ to make JyersUI work on an SKR mini E3 board through a re-mapping of the screen cable to different pins on the board. I feel like I'm close to understanding the how to make it work but my root question is this.

### Is it worth it to go through the steps to run JyersUI on the SKR mini E3 board?

I got my board as a kit with my BLTouch and I am very new to 3d printing. My Ender 3 V2 is my first printer and after about 2 weeks I've finally gotten to the point of reliably being able to get prints to stick (painter's tape gang I guess?). I'm still running stock firmware and through my research I can absolutely see the benefits of JyersUI but the placement of my printer in my house dictates me working to make it as quiet as possible and something the SKR mini E3 board is supposed to help with was getting the stepper motors to run silently because it uses a different stepper motor controller. There's also a think about saving print settings to the board vs saving to the card. Is that supported on the regular board with just Jyers?

I get that flashing on JyersUI is an easy process and will be a marked improvement over the bundled Marlin that came with the printer. I'm just trying to figure out if the juice is worth the squeeze to go through the potential frustration doing the board change and then installing JyersUI on top of it. Like what's the differences in the firmware between what Bigtree bundles with their board and what Jyers is able to accomplish?

Thank you

**Comment by RipperDrone** [comment]
You may want to read my other post, almost there...

**Comment by RipperDrone** [comment, marked as answer]
However, stock Ender steppers are silent drive motors already, E3 won't make them more silent. Noisy fans are your enemies, check out thingiverse and the internet for silent modding them. This was the biggest progress I found in noise level.
